app/assets/javascripts/controllers/ansible_repository/repository_form_controller.js

```diff
@@ -0,0 +1,114 @@
+ManageIQ.angular.app.controller('repositoryFormController', ['$scope', 'repositoryId', 'miqService', 'API', function($scope, repositoryId,  miqService, API) {
+  var vm = this;
+
+  var init = function() {
+    vm.afterGet = false;
+
+    vm.repositoryModel = {
+      name: '',
+      description: '',
+      scm_type: 'git',
+      manager_resource: {},
+      scm_url: '',
+      scm_credentials: null,
```

Comment:
@jameswnl there are no stupid questions :) I must have missed some change because `scm_credentials` no longer exists and there's `authentication_id` instead. Thanks :)